hazelcast-client/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -1014,39 +1015,20 @@ public QueryCacheConfig getOrNullQueryCacheConfig(String mapName, String cacheNa
         return lookupByPattern(configPatternMatcher, queryCacheConfigsForMap, cacheName);
     }
 
-
     /**
-     * Sets attribute to client to be used on the server side.
-     * Attributes can be accessed from {@link com.hazelcast.core.Client}
+     * Adds a label for this client available
      *
-     * @param key   the key of the attribute
-     * @param value the value of the attribute
-     * @return configured {@link com.hazelcast.client.config.ClientConfig} for chaining
+     * @param label The label to be added.
      */
-    public ClientConfig setAttribute(String key, String value) {
-        attributes.put(key, value);
-        return this;
+    public void addLabel(String label) {
```

Comment:
Can you put ClientConfig as a return value? Also please mention  {@link com.hazelcast.core.Client} on the javadoc.